ui/lib/kv/addon/components/page/list.hbs

```diff
@@ -121,6 +123,17 @@
         </div>
       </LinkedBlock>
     {{/each}}
+    {{! Pagination }}
+    {{#if (gt @meta.lastPage 1)}}
```

Comment:
Wrapping pagination in this if block means that if I have, say, 35 items and I change the page size to 50, the pagination will go away once it transitions because there will only be 1 page at that page size. I think instead we should just always show the pagination

Reply:
I went back and forth on this. This is the same conditional block we use now, and because we're not allowing them to change the pageSize via the component in this release (I set `showSizeSelector=false`) I figured keep the current pattern. But just you raising the question is enough nudge for me to remove.

Here's the old conditional block for reference:
```
{{#if (gt @meta.lastPage 1)}}
      <Hds::Pagination::Numbered
        @currentPage={{@meta.currentPage}}
        @currentPageSize={{@meta.pageSize}}
        @route={{@routeName}}
        @showSizeSelector={{false}}
        @totalItems={{@meta.total}}
        @queryFunction={{this.paginationQueryParams}}
        data-test-pagination
      />
    {{/if}}
```

Reply:
And here is what it will look like if we have one secret. 
![image](https://github.com/hashicorp/vault/assets/6618863/9ce9baa5-a4bd-4543-9650-d2f8539254eb)

Reply:
Ah, thanks for that context! I missed `showSizeSelector=false` on first review. If it can't be changed hiding it is fine, but the pagination is pretty enough I don't mind it on the page either 😄